common/utils.go

```diff
@@ -144,3 +145,23 @@ func GetBridgeNetDev(bridgeName string) ([]NetDev, error) {
 		return link.Attrs().Name == bridgeName
 	})
 }
+
+// Configure the ARP cache parameters for the given interface.  This
+// makes containers react more quickly to a change in the MAC address
+// associated with an IP address.
+func ConfigureArpCache(ifname string) error {
```

Comment:
I made a similar function here https://github.com/weaveworks/weave/pull/1958/commits/0c3f751b492f4991b33887ba9c710da978916063, that does not rely on `ioutil.WriteFile` because it uses O_CREATE and hence won't error if the path does not exist